app/globals.css

```
@@ -86,4 +86,52 @@
+
+  .nexus-focus-ring {
+    @apply focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-indigo-500 focus-visible:ring-offset-2 focus-visible:ring-offset-slate-50 dark:focus-visible:ring-offset-[#08090a];
+  }
+
+  .nexus-scrim {
+    @apply fixed inset-0 bg-slate-950/55 backdrop-blur-[2px];
+  }
+
+  .nexus-bottom-sheet {
+    @apply fixed inset-x-0 bottom-0 mx-auto max-h-[88dvh] overflow-y-auto rounded-t-[28px] bg-white px-4 pt-3 shadow-2xl dark:bg-[#151617] sm:max-w-160 sm:px-6;
+    padding-bottom: max(1rem, env(safe-area-inset-bottom));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)